spec-old-api/03-plugins/25-response-rate-limiting/02-policies_spec.lua

```diff
@@ -13,10 +13,10 @@ describe("Plugin: response-ratelimiting (policies)", function()
     local dao
 
     setup(function()
-      dao = select(3, helpers.get_db_utils())
+      local _, db
+      _, db, dao = helpers.get_db_utils("postgres", {})
```

Comment:
is this test meant to be postgres-only?

Reply:
It was implicitly pg-only (when `get_db_utils` gets no parameters it uses the one that's on the test config file, which is pg). I made it explicit while trying to debug why the tests were failing on my machine. I have undone this change (notice this is on `spec-old-api`, so no big deal).